.travis.yml

```diff
@@ -63,27 +62,31 @@ notifications:
     skip_join: true
 
 before_deploy:
- - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+ - if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
      wget https://mirrors.edge.kernel.org/ubuntu/pool/universe/n/nsis/nsis-common_3.04-1_all.deb;
      wget https://mirrors.edge.kernel.org/ubuntu/pool/universe/n/nsis/nsis_3.04-1_amd64.deb;
      sudo dpkg -i nsis-common_3.04-1_all.deb;
      sudo dpkg -i nsis_3.04-1_amd64.deb;
-     cd packaging && ./update-wiki.sh ${TRAVIS_TAG} && cd ..;
-   fi;
- - export PATH=$PATH:$HOME/usr/bin
+     echo ${TRAVIS_REPO_SLUG};
+     if [[ "${TRAVIS_REPO_SLUG}" == "OpenRA/OpenRA" ]]; then
+       cd packaging && ./update-wiki.sh ${TRAVIS_TAG} && cd ..;
+     fi;
+   fi
+ - export PATH=${PATH}:${HOME}/usr/bin
  - DOTVERSION=`echo ${TRAVIS_TAG} | sed "s/-/\\./g"`
  - cd packaging
  - mkdir build
  - ./package-all.sh ${TRAVIS_TAG} ${PWD}/build/
- - cd itch && ./upload.sh ${TRAVIS_TAG} ${PWD}/build/
+ - if [[ "${TRAVIS_REPO_SLUG}" == "OpenRA/OpenRA" ]]; then
+     ./upload-itch.sh ${TRAVIS_TAG} ${PWD}/build/;
+   fi
+
 deploy:
   provider: releases
-  api_key:
-    secure: "g/LU11f+mjqv+lj0sR1UliHwogXL4ofJUwoG5Dbqlvdf5UTLWytw/OWSCv8RGyuh10miyWeaoqHh1cn2C1IFhUEqN1sSeKKKOWOTvJ2FR5mzi9uH3d/MOBzG5icQ7Qh0fZ1YPz5RaJJhYu6bmfvA/1gD49GoaX2kxQL4J5cEBgg="
+  token: ${GH_DEPLOY_API_KEY}
```

Comment:
This will need to be set up in the @orabot account, which @Mailaender has the 2fa for.

Reply:
Done.